### ISSUE TITLE
Accessible Search Input with or without Twitter Typeahead

### ISSUE THREAD

**Issue by mtribone** [psu-libraries/cho#536]
The search input caused some confusion with our accessibility testers. Blacklight has two inputs right next to each other in the code and visually stacked in the UI. Explore alternative approach and override the Blacklight implementation.  [See notes for details](https://psu.box.com/s/cn2xb0agw8oebun4zmxxg4dnq4pyp3n9).
